### PR TITLE
CLC-5752 - Recent Activity: item title can overlap the toggle

### PR DESCRIPTION
**Pull request by davidlimcheng**
Submitted with design consideration and approval from Rachel Hollowgrass.
https://jira.ets.berkeley.edu/jira/browse/CLC-5752

### CODE REVIEW

File: src/assets/templates/widgets/activity_list.html [hunk 1]
```diff
@@ -9,7 +9,9 @@
           <div class="cc-widget-activities-text-column">
             <i class="fa fa-{{typeToIcon[activity.type]}} cc-left"></i>
             <div class="cc-widget-activities-text-margin">
+              <div class="cc-widget-activities-title">
               <strong>{{activity.title}}</strong>
```
Comment (christianvuerings) [inline src/assets/templates/widgets/activity_list.html]: @davidlimcheng could we apply the `cc-widget-activities-title` class to the `strong` tag instead and see whether it still works?

**Comment by christianvuerings** [comment]
@davidlimcheng could you make sure the format of your PR is correct?

* Jira number & title in the title of the PR
* Jira link in the description of the PR

Have a look at https://github.com/ets-berkeley-edu/calcentral/pull/4589 and feel free to make the changes on this PR.

**Comment by davidlimcheng** [comment]
@christianvuerings my bad, should be good now!

**Comment by christianvuerings** [comment]
@davidlimcheng looks great, thanks - we should be good to merge as soon as you make the `<strong class` fix

**Comment by davidlimcheng** [comment]
Thanks @christianvuerings , that makes a lot of sense.  Let me know if this is OK - I'll definitely put future PRs in the same format.

**Comment by christianvuerings** [comment]
Lgtm @johncrossman would you be ok with merging this just after the QA tagging this afternoon?

@davidlimcheng it's also handy if you add the jira number in the beginning of every commit. (you don't have to change it for this PR though)

**Comment by johncrossman** [comment]
@christianvuerings Not a problem to merge *after* qa branch is cut.

**Comment by davidlimcheng** [comment]
@johncrossman @christianvuerings thanks guys!  Wish I could frame this.....digital moment.  My first bug fix!

**Comment by johncrossman** [comment]
The qa branch was cut. Your inaugural merge happens now!

**Pull request merged**